app/controllers/books/comments_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Books
+  class CommentsController < ApplicationController
+    before_action :set_book
+    before_action :set_comment, only: %i[edit update destroy]
+    before_action :set_commentable
+
+    def create
+      @comment = current_user.comments.build(comment_params.merge(commentable: @book))
```

Comment:
```suggestion
      @comment = current_user.comments.build(commentable: @book, **comment_params)
```

これでも同じ結果になるかもです。（動作未検証ですが）

Reply:
ご指摘ありがとうございます！キーワード引数を用いる方がコードが読みやすくなりますね💦 
確認したところ、正常に動作しました😄